### PR TITLE
added feature to share/copy story link

### PR DESCRIPTION
**Pull request by mharshita**
fixes: #23 

Feature added to share on social media platforms or copy the link of the story page.  I have added two options. One is to share on social media and other is to copy the URL of page.
I have taken example of eos-icons.com for LinkedIn, facebook and whatsapp template example.

https://user-images.githubusercontent.com/60281688/122768836-9c6f6800-d2c1-11eb-8510-03bd1ab60967.mp4

image
image
image

### CODE REVIEW

File: src/pages/Story.js [hunk 1]
```diff
@@ -184,12 +215,22 @@ const Story = (props) => {
...
-                  </Button>
+                  <>
+                    <Button
+                      className='btn btn-default'
+                      onClick={() => setEditor(true)}
+                    >
+                      Edit
+                    </Button>{' '}
+                    &nbsp; &nbsp;
+                    <object className='share-story' onClick={togglePopup}>
```
Comment (kbhutani0001) [inline src/pages/Story.js:226]: use css whereverf possible &nbsp is not a good way of adding space

File: src/pages/Story.js [hunk 2]
```diff
@@ -184,12 +215,22 @@ const Story = (props) => {
...
+                      <i className='eos-icons share-story'> share </i>
+                    </object>{' '}
+                    &nbsp; &nbsp;
+                    <object className='share-story' onClick={copy}>
```
Comment (kbhutani0001) [inline src/pages/Story.js:230]: what's this object btw? I have never used it 👀

Comment (mharshita) [reply src/pages/Story.js:230]: Object tag is actually used for external resources. Actually I wanted to make these two icons clickable and at the same time wanted them to be in the same div as that of button that's why I used this. There must be a better way to do this.

Comment (mharshita) [reply src/pages/Story.js]: I have changed the object to transparent button :)

**Pull request merged**